TreeDB/internal/valuelog/block_compression_parts_test.go

```
@@ -0,0 +1,135 @@
+		_ = w.Close()
+		t.Fatalf("eligible parts path staged payload copies: raw=%d encoded=%d", cap(w.rawScratch), cap(w.blockScratch))
+	}
+	if err := w.Close(); err != nil {
+		t.Fatal(err)
+	}
+
+	f, err := os.Open(path)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer f.Close()
```

golangci-lint (2.13.2)

[error] Error return value of `f.Close` is not checked

(errcheck)